tuxemon/db.py

```diff
@@ -451,6 +456,18 @@ def translation_exists_category(cls: MonsterModel, v: Any) -> Any:
             return v
         raise ValueError(f"no translation exists with msgid: {v}")
 
+    @field_validator("catch_rate")
+    def check_catch_rate(cls: MonsterModel, v: Any) -> Any:
+        if 0 <= v <= 255:
+            return v
+        raise ValueError(f"the catch rate is between 1 and 255 ({v})")
```

Comment:
confused by this.  the test will allow 0 and values less that 1, but the error message states the value must be between 1-255, so 0-1 shouldn't be allowed.  idk if the message or the test needs to change here

Reply:
also the type for v should be a float, im assuming, not Any (same with the return type)

Reply:
0,255 is correct, I forgot to change the number
v should be a float